### PR TITLE
Document and clean FutarchyChallenge etc

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

**Comment by alexfigtree** [comment]
@return would be good where needed!

**Comment by jpkcambridge** [comment]
Looks good to me.

File: contracts/Challenge/FutarchyChallenge.sol [hunk 1]
```diff
@@ -7,29 +7,42 @@ import "zeppelin/math/SafeMath.sol";
 import "./ChallengeInterface.sol";
 
 contract  FutarchyChallenge is ChallengeInterface {
+
   // ============
-  // STATE VARIABLES:
+  // STATE:
   // ============
+  // GLOBAL VARIABLES
+  address challenger;     // the address of the challenger
```
Comment (mikec) [inline contracts/Challenge/FutarchyChallenge.sol]: might as well make all the globals `public`, right?

File: contracts/Challenge/FutarchyChallenge.sol [hunk 2]
```diff
@@ -62,24 +84,27 @@ contract  FutarchyChallenge is ChallengeInterface {
...
 
+  /// @dev ended  returns whether Challenge has ended
   function ended() public view returns (bool) {
     return futarchyOracle.isOutcomeSet();
   }
 
+  /// @dev passed  returns whether Challenge has passed
   function passed() public view returns (bool) {
-    require(futarchyOracle.isOutcomeSet());
+    require(this.ended());
```
Comment (mikec) [inline contracts/Challenge/FutarchyChallenge.sol]: not sure that the `this.` is needed

**Pull request merged**